src/core/Table/TableRowMemoized.tsx

```diff
@@ -48,12 +54,26 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const expandedHeight = React.useRef(0);
 
+  const mergedProps = {
+    ...row.getRowProps({
+      className: cx('iui-row', {
+        'iui-selected': row.isSelected,
+        'iui-row-expanded': row.isExpanded && subComponent,
+        'iui-disabled': isRowDisabled,
+      }),
+    }),
+    ...rowProps?.(row.original),
```

Comment:
Wont this override all our classes, if there is a class passed?

Reply:
Good catch! Fixed.